server/views/licenceConditions/standard.pug

```diff
@@ -19,6 +20,52 @@ block content
         for condition in conditions
           li.midPaddingBottom #{condition.text}
 
+
+  h2.heading-large Additional conditions
+  div.pure-u-1.pure-u-md-2-3
+    - var results = false
+    ol.numberedList
+      for condition in licenceConditions
+          if condition.group !== undefined && condition.group !== 'Post-sentence supervision only' && condition.group !== 'Bespoke'
```

Comment:
unnecessary undefined check?
Might be nicer to move this logic out of the template? - passing the types of conditions down as separate collections..

Reply:
The undefined check was necessary because the user could continue to subsequent pages even if they didn't select yes or no 
Without checking for undefined would have meant the user seeing a bespoke condition that wasn't explicitly approved. 
Re 'passing the types of conditions down as separate collections..' I started down that path but decided against it because other pages have used the approach of sending everything to the front

Reply:
amended